src/content/en/2022/third-parties.md

```diff
@@ -1,22 +1,400 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Third Parties
-#TODO - Review and update chapter description
-description: Third Parties chapter of the 2022 Web Almanac covering data of what third parties are used, what they are used for, a deep dive into performance impacts and a discussion on security and privacy impacts.
+description: Third Parties chapter of the 2022 Web Almanac covering data of what third parties are usedon the web and a deep dive into preventing them causing negatibe performance impacts.
 authors: [imeugenia]
-reviewers: [tunetheweb, housseindjirdeh, pepelsbey, kevinfarrugia, alexnj]
-analysts: [kevinfarrugia, msolercanals]
-editors: []
+reviewers: [tunetheweb, kevinfarrugia, alexnj]
+analysts: [kevinfarrugia]
+editors: [shantsis]
 translators: []
+imeugenia_bio: Eugenia is a Senior frontend engineer at <a hreflang="en" href="https://gorillas.io/">Gorillas</a>. She is passionate about web performance and state machines.
```

Comment:
@imeugenia I filled in this bio for you, but feel free to update it to whatever you want.

Reply:
Have updates my bio, thank!